as-controller-board-status/main.go

```diff
@@ -18,66 +17,68 @@ const (
 )
 
 func main() {
-	// Create an instance of the EdgeX SDK and initialize it
-	edgexSdk := &appsdk.AppFunctionsSDK{ServiceKey: serviceKey}
-	err := edgexSdk.Initialize()
-	if err != nil {
-		edgexSdk.LoggingClient.Error(fmt.Sprintf("SDK initialization failed: %v", err.Error()))
-		os.Exit(-1)
+	// See https://docs.edgexfoundry.org/2.2/microservices/application/ApplicationServices/
+	//       for documentation on application services.
+	service, ok := pkg.NewAppService(serviceKey)
+	if !ok {
+		os.Exit(1)
 	}
 
+	lc := service.LoggingClient()
+
 	// Get the application's settings from the configuration.toml file
-	appSettings := edgexSdk.ApplicationSettings()
+	appSettings := service.ApplicationSettings()
 	if appSettings == nil {
-		edgexSdk.LoggingClient.Error("No application settings found")
-		os.Exit(-1)
+		lc.Error("No application settings found")
+		os.Exit(1)
 	}
 
 	boardStatus := functions.CheckBoardStatus{
 		DoorClosed:    true, // Set default door state to closed
 		Configuration: new(functions.ControllerBoardStatusAppSettings),
+		Service:       service,
 	}
 
 	// Retrieve & parse the required application settings into a proper
 	// configuration struct
-	err = utilities.MarshalSettings(appSettings, boardStatus.Configuration, false)
+	err := utilities.MarshalSettings(appSettings, boardStatus.Configuration, false)
```

Comment:
This is an opportunity to use custom configuration. Separate PR.

Reply:
yes, this is an issue added, I will work on it next